app/design/frontend/Magento/blank/Magento_Checkout/web/css/source/module/_cart.less

```diff
@@ -346,6 +346,10 @@
 
         .widget {
             float: left;
+            
```

Comment:
Please remove the blank line.

Reply:
Hey @niravkrish, look at the line 308:
https://github.com/magento/magento2/blob/c4a3e5158af088b9735ff8c2abdd881960b344ae/app/design/frontend/Magento/blank/Magento_Checkout/web/css/source/module/_cart.less#L305-L309

It is better to separate to have a clear vision of what's happening

Reply:
You are right, @LucasCalazans 

Look this commit
https://github.com/magento/magento2/commit/8b12310eaa4d333f78e81c75b2f5cb0a76598049#diff-9d67e88ad584c2949ec42d5200ecf6d7